yarn-project/simulator/src/avm/opcodes/conversion.ts

```diff
@@ -0,0 +1,56 @@
+import { type AvmContext } from '../avm_context.js';
+import { TypeTag, Uint8 } from '../avm_memory_types.js';
+import { Opcode, OperandType } from '../serialization/instruction_serialization.js';
+import { Addressing } from './addressing_mode.js';
+import { Instruction } from './instruction.js';
+
+export class ToRadixLE extends Instruction {
+  static type: string = 'TORADIXLE';
+  static readonly opcode: Opcode = Opcode.TORADIXLE;
+
+  // Informs (de)serialization. See Instruction.deserialize.
+  static readonly wireFormat: OperandType[] = [
+    OperandType.UINT8, // Opcode
+    OperandType.UINT8, // Indirect
+    OperandType.UINT32, // src memory address
+    OperandType.UINT32, // dst memory address
+    OperandType.UINT32, // radix (immediate)
+    OperandType.UINT32, // number of limbs (Immediate)
+  ];
+
+  constructor(
+    private indirect: number,
+    private srcOffset: number,
+    private dstOffset: number,
+    private radix: number,
+    private numLimbs: number,
+  ) {
+    super();
```

Comment:
do you want to assert that the radix is < 255 or 256? because it looks like the output will be a byte.

Reply:
Yep this is sensible, it will probably be easier for us to handle if it is a U8 constrained as well. The noir function calls accept radix values of U32 but indeed return bytes. Not sure how they plan to handle this contradiction on their end.